VikingQuiz.UI/src/Components/YesNoComponent/YesNoComponent.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import './YesNoComponent.css';
+
+
+class YesNoComponent extends React.Component<any, any>{
+
+    constructor(props: any) {
+        super(props);
+  
+        this.state = {
+            serverMessage: ''
```

Comment:
"What is my purpose?"

Since you can remove this component's state completely, you can convert it into a stateless function (rather than a class)